src/Screens/ControlsScreen.cpp

```diff
@@ -0,0 +1,185 @@
+#include "Screens/ControlsScreen.hpp"
+
+ControlsScreen::ControlsScreen(){
+    //background
+    if (!background.loadFromFile("../resource/background/start_background.jpg"))
+    {
+        // error...
+    }
+    bg.setTexture(background);
+
+    //logo
+
+    if (!logo.loadFromFile("../resource/API.png"))
+    {
+        // error...
+    }
+    logo_sprite.setTexture(logo);
+    logo_sprite.setScale(LOGO_SCALE,LOGO_SCALE);
+    logo_sprite.setPosition(SCREEN_WIDTH/2 - 476*LOGO_SCALE,50);
+
+
+    //tagline
+
+    if (!textFont.loadFromFile("../resource/fonts/Adventure.otf"))
+    {
+        // error...
+    }
+    tagline.setFont(textFont);
+
+    tagline.setString("Rebind Controls");
```

Comment:
All these lines to do basically the same thing a couple times... maybe we could have a function that takes font, string, color, size, and position as args, and deals with most of setting up these text objects